utils.sh

```diff
@@ -21,3 +21,65 @@ for patch in "${PWD}/ignition_patches/${kind}/"*.json; do
 done
 sudo cp "$current" "$target"
 }
+
+function dom_mac_retrieve() {
```

Comment:
Minor nit - would it be cleaner is we just made this a seperate script vs inlined?  We could create a uils directory, which could container a bash wrapper and any other tools (including python scripts etc)?

Reply:
I would like to keep it here for now and as soon as I land the PR for adding the second network I will move the scripts.